### PR TITLE
Add a new volume that preserves the history

### PR DESCRIPTION
**Pull request by pradumangoyal**

### CODE REVIEW

**Comment by pradumangoyal** [comment]
I am not sure about the practices. Suggest if there are any changes.

**Comment by pradumangoyal** [comment]
Should we also preserve the bash history?

**Comment by dhruvkb** [comment]
@pradumangoyal this was much needed, thanks! Yes, please also store bash history in the same volume, so name is something generic like `history` instead of being IPython specific.

**Comment by pradumangoyal** [comment]
@dhruvkb Kindly check. Also, note the https://github.com/IMGIITRoorkee/omniport-backend/pull/73 with this PR.

File: django/Dockerfile [hunk 1]
```diff
@@ -76,6 +80,8 @@ RUN chown django:django ./supervisord.conf \
 
 # Define environment variables
 ENV PYTHONPATH="/omniport:/omniport/core:/omniport/services:/omniport/apps"
+ENV HISTFILE="/.history/.bashrc"
```
Comment (dhruvkb) [inline django/Dockerfile]: I think history is stored in `.bash_history` not `.bashrc`. Please confirm.

Comment (pradumangoyal) [reply django/Dockerfile]: Oh, I missed that. Yeah it works it's just any file, but I will push it after the change.

**Comment by pradumangoyal** [comment]
@dhruvkb changed!

**Comment by shaddygarg** [comment]
Much needed!! :heart:

**Pull request merged**